source/os/src/osSysinfo.c

```diff
@@ -1007,6 +1007,11 @@ SysNameInfo taosGetSysNameInfo() {
     tstrncpy(info.machine, uts.machine, sizeof(info.machine));
   }
 
+  char     localHostName[512];
+  taosGetlocalhostname(localHostName, 512);
+  TdCmdPtr pCmd = taosOpenCmd("scutil --get LocalHostName");
```

Comment:
is there any interface to call for getting localhostname instead of calling external command?